ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -158,6 +161,30 @@ tests:
     env:
       GATEWAY_MODE: local
     workflow: openshift-e2e-aws-ovn-local-to-shared-gateway-mode-migration
+- as: unit
+  commands: |
+    cd go-controller
+    PKGS=$(go list -mod vendor -f '{{if len .TestGoFiles}} {{.ImportPath}} {{end}}' ${PKGS:-./cmd/... ./pkg/... ./hybrid-overlay/...} | xargs)
+    # these packages need sudo access, skip them for now
```

Comment:
I pushed https://github.com/ovn-org/ovn-kubernetes/pull/2932

which will allow us to just skip the tests that need the access. With that PR you can just do make test NOROOT=TRUE and it will skip them automatically.

Reply:
Ok, do we want to wait on that PR to land. Can we just commit this one, and then update the config when that PR merges downstream?